Stripe/STPPaymentOptionsInternalViewController.m

```diff
@@ -166,6 +169,31 @@ - (BOOL)isPaymentOptionDetachable:(id<STPPaymentOption>)paymentOption {
     return YES;
 }
 
+- (NSArray<id<STPPaymentOption>> *)cardPaymentOptions {
+    return [self.paymentOptions filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(id<STPPaymentOption> _Nullable evaluatedObject, NSDictionary<NSString *,id> * __unused _Nullable bindings) {
+        if ([evaluatedObject isKindOfClass:[STPPaymentMethodParams class]]) {
+            STPPaymentMethodParams *paymentMethodParams = (STPPaymentMethodParams *)evaluatedObject;
+            if (paymentMethodParams.type == STPPaymentMethodTypeFPX) {
```

Comment:
can this be like `if (paymentMethodParams.type != STPPaymentMethodTypeCard)` so we don't accidentally regress it when adding a new type'?

Reply:
Sure, that'd be more forward-thinking. (Though a regression here would be very visible!)